src/PowerShellEditorServices/Session/Host/TerminalPSHostRawUserInterface.cs

```diff
@@ -263,11 +286,21 @@ public override void FlushInputBuffer()
             Coordinates origin,
             BufferCell[,] contents)
         {
-            Logger.Write(
-                LogLevel.Warning,
-                "PSHostRawUserInterface.SetBufferContents was called");
+            this.internalRawUI.SetBufferContents(origin, contents);
         }
 
         #endregion
+
+        private static bool IsCtrlC(ConsoleKeyInfo keyInfo)
+        {
+            // In the VSCode terminal Ctrl C is processed as virtual key code "3", which
+            // is not a named value in the ConsoleKey enum.
+            if ((int)keyInfo.Key == 3)
```

Comment:
so odd. Probably should test this on other platforms too just in case.

Reply:
I would bet it's the same.  `winpty` seems to lean more towards unix-like behavior when it comes to input.  Definitely worth testing though.